### PR TITLE
osci3617: respect resultOutcome

### PR DESCRIPTION
**Pull request by Andrei-Stepanov**
Signed-off-by: Andrei Stepanov <astepano@redhat.com>

### CODE REVIEW

File: src/components/ArtifactGreenwaveKaiState.tsx [hunk 1]
```diff
@@ -122,7 +122,7 @@ export const FaceForGreenwaveKaiState: React.FC<
     const resultOutcome = state.gs.result?.outcome;
     const requimentType = state.gs.requirement?.type;
     // Prefer Greenwave's point of view over status from UMB message.
-    const iconName = requimentType || resultOutcome || 'unknown';
+    const iconName = resultOutcome || requimentType || 'unknown';
```
Comment (jimbair) [inline src/components/ArtifactGreenwaveKaiState.tsx:125]: I believe this should be 'requirementType', though we would want to do an s/requimentType/requirementType/g to grab any other typos.

Comment (Andrei-Stepanov) [reply src/components/ArtifactGreenwaveKaiState.tsx]: @jimbair thank you, updated.

**Comment by mgrabovsky** [comment]
Hm, I'm not sure that this correct, i.e. that it doesn't reintroduce the bug fixed in #26. Can you please verify that after this change, the red cross icon, for example, is displayed for a required test which has the `NEEDS_INSPECTION` outcome?

**Comment by mgrabovsky** [comment]
Yeah, this breaks that

Screenshot 2022-08-18 at 13-00-19 ❌ kernel-4 18 0-372 24 1 el8_6 CI Dashboard
:

**Comment by Andrei-Stepanov** [comment]
@mgrabovsky  I updated the logic for icon. Please rewiev.

**Review by mgrabovsky — APPROVED** [review]
Looks good to me. I've changed the wording of comments slightly in order to make it more precise what's going on.

**Pull request merged**